specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -3162,6 +3170,206 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/addons": {
+      "get": {
+        "operationId": "Addons_List",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "tags": [
+          "Addons"
+        ],
+        "summary": "List addons in a private cloud",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/privateCloudName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/AddonList"
```

Comment:
you are exposing new api functionality, this should be modeled as a new api-version, see the bot's recommendation as well (NewApiVersionRequired)

Reply:
It was in this version in the pull request for the private repo. It was left out by mistake. It is already being used in the portal UI in production. Please see my [earlier comments.](https://github.com/Azure/azure-rest-api-specs/pull/12236#issuecomment-751894356). The easiest solution is to add the changes back in to this API version.